lib/widgets/scrollbar.dart

```diff
@@ -1,29 +1,12 @@
 /*---------------------------------------------------------------------------------------------
 *  Copyright (c) nt4f04und. All rights reserved.
 *  Licensed under the BSD-style license. See LICENSE in the project root for license information.
-*
-*  Copyright (c) The Flutter Authors.
-*  See ThirdPartyNotices.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
```

Comment:
Do we still want the copyright header?

Reply:
No, it's no longer needed - I only put it in files that contain code from external sources